operators/vtk_renderer/README.md

```diff
@@ -0,0 +1,18 @@
+### vtk_renderer operator
+
+The `vtk_renderer` extension takes the output the source video player and the
+output of the `tool_tracking_postprocessor` operator and renders the video
+stream with an overlay annotation of the label of the tool being used.
+
```

Comment:
Please briefly note performance characteristics of the operator, including how often it performs host/GPU copies and any relevant benchmarking information. You can copy/paste what you've written in the PR description.